CONVENTIONS.md

```diff
@@ -29,6 +31,125 @@ Black is left at defaults, flake8 and isort are configured to adhere to black.
 
 All functions need to be covered by unit tests.
 
+# Coordinate frame conventions
+
+The primary purpose of pylinalg as a library is to support linear algebra
+operations in pygfx and applications based on pygfx. As such, pylinalg shares
+its coordinate frame conventions with pygfx, which in turn follows [gITF's
+conventions
+](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#coordinate-system-and-units).
+
+## Coordinate Systems
+
+Unless stated otherwise, frames use 3-dimensional euclidean coordinates and a
+right-handed coordinate frame. What differs depending on the type are the
+semantics used to describe each axis. Distances are measured in `m` (meters) and
+and angles in `rad` (radians) unless stated otherwise.
+
+* **Object Coordinates** are represented using `(x, y, z)` vectors and are used
+  to describe objects in a scene. They use the following semantics:
+  * The *negative* X axis is called *right*.
+  * The positive Y axis is called *up*.
+  * The positive Z axis is called *forward*.
+* **Camera Coordinates** are represented using `(x, y, z)` vectors and are used
```

Comment:
This also applies to Lights in glTF. See here: https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_lights_punctual#adding-light-instances-to-nodes

Reply:
Hm, I just thought that lights point along (0, 0, -Z) but are otherwise the same as normal objects. I.e., if I move a light "forward" by 2 units, will it travel along positive or negative Z?

Reply:
I think that's what is meant by

> the light's direction is defined as the 3-vector `(0.0, 0.0, -1.0)`

Reply:
> It's probably because shadow casting requires rendering the scene from the light's point of view, as if it were a camera.

Aha! 💡 

I think both camera and light shine/look forward, which is their `(0, 0, -1)` axis.

Reply:
Then I will update to include lights as using camera coordinates.